_opensearch/segment-replication/configuration.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Segment Replication Configuration
+nav_order: 122
+parent: Segment Replication Feature
+grand_parent: Opensearch
+---
+
+## Segment Replication Configuration
+
+To enable the segment replication type, reference the steps below.
+
+### Enabling the feature flag
+
+There are several methods to enable segment replication depending on install type. You will also need to set replication strategy to `SEGMENT` when creating the index.
+
+#### Enable on a node using a tarball install
+
+The flag is toggled using a new jvm parameter that is set in either `OPENSEARCH_JAVA_OPTS` or in config/jvm.options.
+
+1. Option 1. Update config/jvm.options with:
+
+    ````json
+    -Dopensearch.experimental.feature.replication_type.enabled=true
+    ````
+
+1. Option 2: Use the `OPENSEARCH_JAVA_OPTS` environment variable:
+
+    ````json
+    export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.replication_type.enabled=true"
+    ````
+1. Option 3: For developers using Gradle, update run.gradle with:
+
+    ````json
+    testClusters {
+      runTask {
+        testDistribution = 'archive'
+        if (numZones > 1) numberOfZones = numZones
+        if (numNodes > 1) numberOfNodes = numNodes
+        systemProperty 'opensearch.experimental.feature.replication_type.enabled', 'true'
+      }
+    }
+    ````
+
+#### Enable with Docker containers
+
+For Docker composer, add the following line to docker-compose.yml underneath the `opensearch-node` and `environment` section:
```

Comment:
Is "Docker composer" the correct term? Or should we say "If you're running Docker, add...."

Reply:
yeah I think that makes more sense as it's a part of Docker :)